specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -3601,6 +3609,283 @@
+              },
+              {
+                "value": "Premium_LRS",
+                "description": "Premium SSD locally redundant storage. Best for production and performance sensitive workloads."
+              },
+              {
+                "value": "StandardSSD_LRS",
+                "description": "Standard SSD locally redundant storage. Best for web servers, lightly used enterprise applications and dev/test."
+              },
+              {
+                "value": "UltraSSD_LRS",
+                "description": "Ultra SSD locally redundant storage. Best for IO-intensive workloads such as SAP HANA, top tier databases (for example, SQL, Oracle), and other transaction-heavy workloads."
+              }
+            ]
+          },
+          "description": "The disk type used by virtual machine in hostpool session host."
+        },
+        "vmCustomConfigurationUri": {
+          "type": "string",
+          "description": "The uri to the storage blob containing scripts to be run on the virtual machine after provisioning."
+        },
+        "imageInfo": {
+          "$ref": "#/definitions/ImageInfoProperties",
+          "description": "Image configurations of HostPool."
+        },
+        "domainInfo": {
+          "$ref": "#/definitions/DomainInfoProperties",
+          "description": "Domain configurations of HostPool."
+        }
+      }
+    },
+    "ImageInfoProperties": {
+      "description": "Image configurations of session host in a HostPool.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Gallery",
+            "StorageBlob",
+            "CustomImage"
+          ],
+          "x-ms-enum": {
+            "name": "ImageType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Gallery",
+                "description": "Using default gallery images offered by azure market place."
+              },
+              {
+                "value": "StorageBlob",
+                "description": "Using a VHD stored in a storage blob."
+              },
+              {
+                "value": "CustomImage",
+                "description": "Using custom image or custom shared image."
+              }
+            ]
+          },
+          "type": "string",
+          "description": "The type of image session hosts use in the hostpool.",
+          "x-nullable": false
+        },
+        "marketPlaceInfo": {
+          "$ref": "#/definitions/MarketPlaceInfoProperties",
+          "description": "The values to uniquely identify a gallery image."
+        },
+        "storageBlobUri": {
+          "type": "string",
+          "description": "The uri to the storage blob which contains the VHD. Image type must be StorageBlob."
+        },
+        "customId": {
+          "type": "string",
+          "description": "The resource id of the custom image or shared image. Image type must be custom."
```

Comment:
"Image type must be custom." should be "Image type must be CustomImage".

Reply:
ok